components/importCSVSection/importCSVSection.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@mui/material'
+import React from 'react'
+import {
+  StyledCsvButton,
+  StyledCsvButtonsContainer,
+  StyledSubHeader
+} from '../../pageStyles/emailSender.styles'
+
+type ImportCSVSectionProps = {
+  file: boolean;
+  handleImportCsv: any;
+  handleUploadCsv: any;
+};
+
+export default function ImportCSVSection ({
+  file,
+  handleImportCsv,
+  handleUploadCsv
+}: ImportCSVSectionProps) {
+  return (
+    <>
+      <StyledSubHeader variant="h5">3) Upload and import csv</StyledSubHeader>
+      <StyledCsvButtonsContainer>
+        <input
+          style={{ display: 'none' }}
+          id="contained-button-file"
+          accept={'.csv'}
+          type="file"
+          onChange={handleUploadCsv}
+        />
+        <label htmlFor="contained-button-file">
+          <Button variant="contained" component="span">
+            Upload
+          </Button>
+        </label>
+        <StyledCsvButton
```

Comment:
StyledCsvButtonsContainer and StyledCsvButton are not used anywhere besides for this component so let's create importCsvSection.styles.ts in the importCSVSection folder. Let's move over all styled components from emailSender.styles.ts that are only used in this component to mportCsvSection.styles.ts

Let's do this for all the components ~